src/config.cc

```diff
@@ -44,6 +44,8 @@ const char *errNotEnableBlobDB = "Must set rocksdb.enable_blob_files to yes firs
 const char *errNotSetLevelCompactionDynamicLevelBytes =
             "Must set rocksdb.level_compaction_dynamic_level_bytes yes first.";
 
+const char *kDefaultBindAddress = "0.0.0.0";
```

Comment:
i think default bind address should be "127.0.0.1" which is safer.

Reply:
@ShooterIT Ok. However, the comment in config file states that: 

`By default kvrocks listens for connections from all the network interfaces available on the server.`

Should I change the related comment?

Reply:
@ShooterIT I changed the default bind address from `0.0.0.0` to `127.0.0.1` and edited the comment in `kvrocks.conf`.

Reply:
thanks @torwig i think old comment is wrong since default bind address is 127.0.0.1, also ping @git-hulk

Reply:
Yes, the default bind address will be better since it can prevent users from starting with external IPs and getting security attacks.